os-eap-launch/added/launch/datasource-common.sh

```diff
@@ -494,7 +494,11 @@ function inject_datasource() {
 
   url=$(find_env "${prefix}_URL")
 
-  case "$db" in
```

Comment:
I don't think this file should have any changes.

Reply:
Sorry, my error, I thought i'd revert those, will update shortly.